projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -22,15 +22,20 @@ export class NgSelectA11yDirective implements AfterViewInit {
     const divCombobox =
       this.elementRef.nativeElement.querySelector('[role="combobox"]');
 
-    this.renderer.setAttribute(
-      divCombobox,
-      'aria-label',
-      this.cxNgSelectA11y.ariaLabel
-    );
-    this.renderer.setAttribute(
-      divCombobox,
-      'aria-controls',
-      this.cxNgSelectA11y.ariaControls
-    );
+    if (this.cxNgSelectA11y.ariaLabel) {
+      this.renderer.setAttribute(
+        divCombobox,
+        'aria-label',
+        this.cxNgSelectA11y.ariaLabel
+      );
+    }
+
+    if (this.cxNgSelectA11y.ariaControls) {
```

Comment:
the same here, but I can see we have so many cases where `aria-controls` is the same as in element, so maybe we can do it optional and use `id` of  `this.elementRef`

Reply:
I find the code more readable when we see the 2 parameters passed like it is at the moment, even though DRY is not strictly applied with element id copied.